docs/INSTALLATION.md

```diff
@@ -0,0 +1,177 @@
+
+    ```html
+    <link rel="stylesheet" href="path/to/node_modules/@clr/icons/clr-icons.min.css">
+
+    <script src="path/to/node_modules/@webcomponents/custom-elements/custom-elements.min.js"></script>
+    <script src="path/to/node_modules/clr-icons/clr-icons.min.js"></script>
+    ```
+
+    If your site is built with [angular-cli](https://github.com/angular/angular-cli) you can achieve the above by adding the files to the styles array and
+    scripts array in `angular-cli.json`:
+
+    ```js
+    "styles": [
+        ...
+        "../node_modules/@clr/icons/clr-icons.min.css",
+        ...
+    ],
+    "scripts": [
+        ...
+        "../node_modules/@webcomponents/custom-elements/custom-elements.min.js",
+        "../node_modules/@clr/icons/clr-icons.min.js"
+        ...
+    ]
+    ```
+
+## Installing Clarity Angular
+
+1.  The easiest way to run a sample Angular application with Clarity is to use
+    the Angular CLI and run `ng add @clr/angular`. If you have an existing project
+    or are not using the Angular CLI, follow the following steps.
+
+2.  Follow the steps above to install Clarity Icons and Clarity UI.
+
+3.  Install the clarity-angular package through npm:
+
+    ```bash
+    npm install @clr/angular --save
+    ```
+
+4.  Import the ClarityModule into your Angular application's module. Your
+    application's main module might look like this:
+
+    ```typescript
+    import { NgModule } from '@angular/core';
+    import { BrowserModule } from '@angular/platform-browser';
+    import { ClarityModule } from '@clr/angular';
+    import { AppComponent } from './app.component';
+
+    @NgModule({
+        imports: [
+            BrowserModule,
+            ClarityModule,
+            ....
+         ],
+         declarations: [ AppComponent ],
+         bootstrap: [ AppComponent ]
+    })
+    export class AppModule {    }
+    ```
+
+    If your application uses [systemjs](https://github.com/systemjs/systemjs),
+    add the configuration as in the example below.
+
+    ```
+    System.config({
+      ...
+      map: {
+         ...
+         '@clr/angular': 'node_modules/@clr/angular/bundles/clr-angular.umd.js',
+         '@clr/icons': 'node_modules/@clr/icons/bundles/clr-icons.umd.js',
+      },
+      ...
+    });
+    ```
+
+## Installing Clarity Web Components
+
+1.  First, install the Clarity Core package from npm.
+
+    ```bash
+    npm install @clr/core --save
+    ```
+
+2.  Import desired Web Component into your JavaScript or TypeScript
+
+    ```typescript
+    import '@clr/core/modal';
+    ```
+
+3.  Use Web Component in desired framework template
+
+    #### Angular
+
+    ```html
+      <!--
+        - `large` is a attribute style hook
+        - [open] is setting a property on the element
+        - (openChange) is listening for the `openChange` custom event
+      -->
+      <cwc-modal large [open]="true" (openChange)="log($event.detail)">
```

Comment:
If we're using hooks like this, I don't think this is a representation that will answer people's questions about turning them on and off.

Reply:
The `large` attr? Yeah I think to your point above it should be `size="lg"` not `large`